### PR TITLE
Desilva/kotlin

### PR DESCRIPTION
**Pull request by tharakadesilva**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,45 +2,42 @@ group 'com.SlackUserPresenceManager'
...
-        from "SlackUserPresenceManager.properties"
-        into "build/libs"
-    }
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+compileKotlin.kotlinOptions.jvmTarget = "1.8"
+compileTestKotlin.kotlinOptions.jvmTarget = "1.8"
+
+repositories {
```
Comment (wcchristian) [inline build.gradle:33]: Just wanted to test a comment in a gh PR

# Testing a header

#6 
Test reference another issue.

Comment (wcchristian) [reply build.gradle:33]: Starting a refview

File: src/main/kotlin/com/slackuserpresencemanager/Main.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    private val PROPERTY_FILE_NAME = "SlackUserPresenceManager.properties"
+
+    private val properties = Properties()
+
+    fun getProperty(key: String): String {
+        return properties.getProperty(key, "")
+    }
+
+    init {
```
Comment (wcchristian) [inline src/main/kotlin/com/slackuserpresencemanager/Main.kt:27]: @tharakadesilva is init just the main method for Kotlin?

**Review by wcchristian — APPROVED** [review]

File: src/main/kotlin/com/slackuserpresencemanager/Main.kt (hunk 2 above)
Comment (tharakadesilva) [reply src/main/kotlin/com/slackuserpresencemanager/Main.kt:27]: @wcchristian init is the constructor

**Pull request merged**